master/internal/experiment_filter.go

```diff
@@ -98,7 +98,7 @@ func expColumnNameToSQL(columnName string) (string, error) {
 		"state":           "e.state",
 		"numTrials":       "(SELECT COUNT(*) FROM trials t WHERE e.id = t.experiment_id)",
 		"progress":        "COALESCE(progress, 0)",
-		"user":            "COALESCE(u.display_name, u.username)",
+		"user":            "e.owner_id",
```

Comment:
Maybe? We're a bit snared because the client's building the filter statement from what comes through the columns api, which maps to the properties on the experiment protobuf, so if we updated this to `userID` we'd probably need a one off exception in the filter code on the client to map `user` to `userID` or update the columns api to specify how to filter each column.